WalletWasabi.Gui/Tabs/WalletManager/RecoverWalletView.xaml

```diff
@@ -29,6 +29,12 @@
               </DataTemplate>
             </ItemsControl.ItemTemplate>
           </ItemsControl>
+          <CheckBox IsChecked="{Binding ShowAdvancedOptions,Mode=TwoWay}"><TextBlock Text="Show advanced options"/></CheckBox>
+          <StackPanel Spacing="8" IsVisible="{Binding ShowAdvancedOptions}">
+            <controls:ExtendedTextBox Text="{Binding AccountKeyPath}" Watermark="Account Key Path" UseFloatingWatermark="True"/>
+            <controls:ExtendedTextBox Text="{Binding MinGapLimit}" Watermark="Number of keys to derive" UseFloatingWatermark="True"/>
```

Comment:
```suggestion
            <controls:ExtendedTextBox Text="{Binding MinGapLimit}" Watermark="Minimum Gap Limit" UseFloatingWatermark="True"/>
```

We occasionally derive more than the minimum gap limit, thus minimum and gap limit, because the "keys to derive" would suggest for me that this many keys would only be derived the first time around.